x-pack/plugin/async-search/src/internalClusterTest/java/org/elasticsearch/xpack/search/CrossClusterAsyncSearchIT.java

```diff
@@ -1538,32 +1729,29 @@ public void testCancelViaAsyncSearchDelete() throws Exception {
                 assertTrue(taskInfo.description(), taskInfo.cancelled());
             }
 
-            ExecutionException e = expectThrows(ExecutionException.class, () -> getAsyncSearch(response.getId()));
-            assertNotNull(e);
-            assertNotNull(e.getCause());
-            Throwable t = ExceptionsHelper.unwrap(e, ResourceNotFoundException.class);
-            assertNotNull("after deletion, getAsyncSearch should throw an Exception with ResourceNotFoundException in the causal chain", t);
+            expectThrows(ResourceNotFoundException.class, () -> getAsyncSearch(response.getId()));
```

Comment:
switching from `get` to `actionGet` simplifies some of the expect throws that I added, plus this one that we already had.